ruby_on_rails/forms_and_authentication/form_basics.md

```diff
@@ -77,7 +77,7 @@ Each one of these inputs is structured slightly differently, but there are some
 
 Will result in your `params` hash containing a key called `description` that you can access as normal, e.g. `params[:description]`, inside your controller.  That's also why some inputs like radio buttons (where `type="radio"`) use the `name` attribute to know which radio buttons should be grouped together such that clicking one of them will unclick the others.  The `name` attribute is surprisingly important!
 
-Now another thing we talked about in the controller section was nesting data.  You'll often want to tuck submitted data neatly into a hash instead of keeping them all at the top level.  This can be useful because, as we saw with controllers, it lets you do a one-line `#create` (once you've whitelisted the parameters with `#require` and `#permit`).  When you access `params[:user]`, it's actually a hash containing all the user's attributes, for instance `{first_name: "foo", last_name: "bar", email: "foo@bar.com"}`.  How do you get your forms to submit parameters like this?  It's easy!
+Now another thing we talked about in the controller section was nesting data.  You'll often want to tuck submitted data neatly into a hash instead of keeping them all at the top level.  This can be useful because, as we saw with controllers, it lets you do a one-line `#create` (once you've allowlisted the parameters with `#require` and `#permit`).  When you access `params[:user]`, it's actually a hash containing all the user's attributes, for instance `{first_name: "foo", last_name: "bar", email: "foo@bar.com"}`.  How do you get your forms to submit parameters like this?  It's easy!
```

Comment:
```suggestion
Now another thing we talked about in the controller section was nesting data.  You'll often want to tuck submitted data neatly into a hash instead of keeping them all at the top level.  This can be useful because, as we saw with controllers, it lets you do a one-line `#create` (once you've allowed the parameters with `#require` and `#permit`).  When you access `params[:user]`, it's actually a hash containing all the user's attributes, for instance `{first_name: "foo", last_name: "bar", email: "foo@bar.com"}`.  How do you get your forms to submit parameters like this?  It's easy!
```

Reply:
Oh, I left those in to preserve the concept of 'Allow Lists' so that learners get exposed to a common business lingo. 
What do you think about leaving those terms in quotes? 🤔

Reply:
I don't think it's that important for students to be exposed to that specific term. By this point in the curriculum, learners should have a good idea about the overall concept of only permitting the parameters they want from the explanation and assignment in the controller lesson. Adding more terms and explanations may be redundant and confusing imo.